lib/nexthop_group.c

```diff
@@ -676,6 +676,47 @@ DEFPY(no_nexthop_group_backup, no_nexthop_group_backup_cmd,
 	return CMD_SUCCESS;
 }
 
+DEFPY(nexthop_group_resilience,
+      nexthop_group_resilience_cmd,
+      "resilient buckets (1-256) idle_timer (1-4294967295) unbalanced_timer (1-4294967295)",
```

Comment:
Not: Can we avoid underscores in command? Maybe replace with a hyphen

Reply:
fixed